tools/create_snapshot_artifact/main.py

```diff
@@ -15,6 +15,8 @@
 sys.path.append(os.path.join(os.getcwd(), "tests"))  # noqa: E402
```

Comment:
Shouldn't this line already ensure that we can import relative to the tests directory?

Reply:
I think this just adds a new path for the python interpreter to look for files from which to do the import but it doesn't actually change the current working directory. The method `get_firecracker_version_from_toml()` expects the working directory to be set to /firecracker/tests, it doesn't look to the paths interpreter uses

Reply:
I think this just adds a new path for the python interpreter to look for files from which to do the import but it doesn't actually change the current working directory. The method get_firecracker_version_from_toml() expects the working directory to be set to /firecracker/tests, it doesn't look to the paths interpreter uses

Reply:
Yes, that allows importing, but it does not move into the `tests` directory. A lot of it just assumes we are in `tests`. A better fix would be to extract the functions that are not related to testing into proper Python modules, and/or to fix the test infra to not assume the test directory. But both are pretty large in scope.

Reply:
Ah, I see, slightly misunderstood what the issue was (thought it was about the imports failing)